OpenMage/magento-lts
# Added support for sending subscribe confirm email

In case "newsletter subscribe confirmation" is enabled, it seems that no "subscription success" email is sent.

This code was provided by @hejhog in issue https://github.com/OpenMage/magento-lts/issues/822, I'm just converting it to a PR, which I've not tested.

### Fixed Issues
1. https://github.com/OpenMage/magento-lts/issues/822

### Manual Testing
1. enable newsletter confirmation
2. subscribe to newsletter and check no success email is sent
